src/main/java/jimmy/mcgymmy/model/food/Fridge.java

```diff
@@ -53,6 +53,18 @@ public void remove(Index removeIndex) {
         internalList.remove(removeIndex.getZeroBased());
     }
 
+    /**
+     * Removes the food item
+     * The food item must exist in the list.
+     */
+    public void remove(Food toRemove) {
+        requireNonNull(toRemove);
+        assert internalList.contains(toRemove) : "Fridge does not contain food item";
```

Comment:
Since you have already had an if statement below, I don't think this line is needed

Reply:
I think it is to confirm that the code is working when we run it with assertions.

The if statement below is to make sure that there are no errors in case we miss some paths / did not run with assertions